javascript/2017-grid-game.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * https://leetcode.com/problems/grid-game/
+ * Linear
+ * Time O(n) | Space O(n)
+ * @param {number[][]} grid
+ * @return {number}
+ */
+
+// Calculate the prefix sums for a given row in the grid
```

Comment:
Remove redundant comments

Reply:
I see someone has added a better solution to this problem [here](https://github.com/neetcode-gh/leetcode/blob/main/javascript/2017-grid-game.js). I'm closing the PR.